networking/p2p/src/test/java/tech/pegasys/artemis/networking/p2p/jvmlibp2p/rpc/encodings/SszEncodingTest.java

```diff
@@ -26,6 +26,7 @@
 class SszEncodingTest {
 
   private final SszEncoding encoding = new SszEncoding();
+  private static final Bytes EXCEEDS_MAXIMUM_LENGTH = Bytes.fromHexString("0x818040");
```

Comment:
It might be good to make it explicit that this hex string is greater than the MAX_CHUNK_SIZE when interpreted as an int.

Reply:
Renamed to `LENGTH_PREFIX_EXCEEDING_MAXIMUM_LENGTH` - there is a comment in the test as well to explain that we should reject based purely on the length prefix without reading the actual data.